src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java

```diff
@@ -187,6 +186,7 @@ public boolean onPreferenceClick(Preference preference) {
       inputField.setHint(R.string.videochat_instance_placeholder);
       inputField.setText(dcContext.getConfig(DcHelper.CONFIG_WEBRTC_INSTANCE));
       inputField.setSelection(inputField.getText().length());
+      inputField.setInputType(TYPE_TEXT_VARIATION_URI);
```

Comment:
this changes only the default keyboard, or?

i am not sure if sth. as `jitsi:https://example.org/?$ROOM` is actually a valid uri.

Reply:
It doesn't matter if the entered string is really a valid URI. The InputType only defines the keyboard layout, independently of the actual content of the EditText view.